modules/apps/document-library/document-library-test/src/testIntegration/java/com/liferay/document/library/webserver/test/WebServerFriendlyURLTest.java

```diff
@@ -69,18 +70,69 @@ public void testHasFilesWithFileEntryFriendlyURLSeparator()
 					_getFileEntryFriendlyURL(urlTitle))));
 	}
 
+	@Test
+	public void testHasFilesWithFileEntryNameHasPlusSign() throws Exception {
+		String nameBase = RandomTestUtil.randomString();
+
+		String fileURL = nameBase + "%2B.txt";
+		String fileName = nameBase + "+.txt";
+
+		Assert.assertFalse(
+			WebServerServlet.hasFiles(
+				_createMockHttpServletRequest(
+					String.format("/%s/0/%s", group.getGroupId(), fileURL))));
+		_addFileEntry(fileName, RandomTestUtil.randomString());
```

Comment:
You need to add a blank line between the assertion and the call to `_addFileEntry`.